e2e/utils/element-utils.ts

```diff
@@ -7,24 +7,24 @@ import {ElementHandle, JSHandle, Page} from 'puppeteer';
  * @param {string} attribute Attribute name.
  */
 export const getAttrValue = async (page: Page, element: ElementHandle, attribute: string): Promise<string> => {
-  return page.evaluate((link, attr) => link.getAttribute(attr), element, attribute);
+  return page.evaluate((link, attr) => {return link.getAttribute(attr)}, element, attribute);
```

Comment:
What does `link` argument name mean in this context? Maybe it should be renamed to `el` for consistency with `attr`.

Am I right in assessing that this function is basically designed to have Puppeteer evaluate the ElementHandle into an actual Element which is then used to return the attribute value?

Reply:
👍 renamed `link` to `elemt` for consistency.
and yes to the second question, the `evaluate` function does just that.